analyticConfigurations/computed_metric-aws.tag.json

```diff
@@ -18,6 +18,13 @@
           "fqn": "netuitive.aws.applicationelb.httpsuccesspercent",
           "name": "HTTP Success Rate"
         }
+      }, {
+        "match": "netuitive.aws.applicationelb.httpnonerrorpercent",
+        "properties": {
+          "cml": "100 * ${aws.applicationelb.httpcode_target_2xx_count}.actual / (${aws.applicationelb.httpcode_target_2xx_count}.actual + ${aws.applicationelb.httpcode_target_5xx_count}.actual)",
+          "fqn": "netuitive.aws.applicationelb.httpnonerrorpercent",
+          "name": "HTTP Non-Error Rate"
```

Comment:
This name is a bit confusing to me. This metric counts the rate of successes versus server errors, leaving out redirection and client errors, right?

Reply:
I had a hard time coming up with a name for this. Any name recommendations are welcome.

Reply:
How about "HTTP Server Errors to Success Ratio"?

Reply:
It's not 5xx/2xx, though, so it's not really a ratio.

Reply:
But it's also not a rate, and it's certainly not measuring HTTP Non-Errors

Reply:
So are we coming to the conclusion that this metric has no place in this package?

Reply:
I'm not sure if that's the conclusion, but I'm still not exactly sure what this metric is measuring. I'm not willing to release it until we know that and can name it appropriately.

Reply:
I agree with @Grumblesmith. I'd also like to know the problem it's trying to solve.